requirements.in

```diff
@@ -25,7 +25,7 @@ psutil>=6.0.0,<7.0.0
 notifications-python-client==10.0.1
 
 # Run `make bump-utils` to update to the latest version
-notifications-utils @ git+https://github.com/alphagov/notifications-utils.git@94.0.1
+notifications-utils @ git+https://github.com/alphagov/notifications-utils.git@95.1.0
```

Comment:
Can we change the version number to `95.1.2` please to get in [this fix](https://github.com/alphagov/notifications-utils/blob/main/CHANGELOG.md#9512)? `95.1.0` gives errors when tasks retry, but it's fixed in that later version

Reply:
This is updated now 🙌🏼